### DISCUSSION TITLE
History and rollback showing old but not latest syncs

### DISCUSSION THREAD

**Discussion by jiribroulik** [argoproj/argo-cd#6815] [Q&A] [answered]
I am running v2.0.4 version and if I click on 'history and rollback' it shows 5 days old deployed versions even though the application is synced 3 times a day, so basically either there is some offset or not sure. Do you know why it is not showing latest deployed sync jobs. This way it is not possible to do proper rollback.

<img width="582" alt="Screen Shot 2021-07-27 at 10 13 15" src="https://user-images.githubusercontent.com/3316782/127120175-fdcbb455-e77a-495a-bc37-766b04740b3c.png">

**Comment by jannfis** [comment, marked as answer]
Do you maybe perform partial/selective syncs? These are not recorded in the history, IIRC same goes for "apply out of sync only" option.

**Reply by jiribroulik** [reply]
You might be right, can you somehow disable partial/selective syncs?

**Reply by jessesuen** [reply]
No, it's not possible to prevent selective syncing.
